src/kbmod/filters/sigma_g_filter.py

```diff
@@ -183,16 +130,45 @@ def compute_clipped_sigma_g_matrix(self, lh):
             A N x T matrix of Booleans indicating if each point is valid (True)
             or has been filtered (False).
         """
-        # We need to
-        os.environ["XLA_PYTHON_CLIENT_PREALLOCATE"] = "false"
-        os.environ["XLA_PYTHON_CLIENT_ALLOCATOR"] = "platform"
+        # Use a GPU if one is available.
+        if torch.cuda.is_available():
+            device = torch.device("cuda")
+        else:
+            device = torch.device("cpu")
```

Comment:
I'm curious if there's a difference in speed between using `jax` and `pytorch` on cpu

Reply:
Yes. On my laptop the pytorch version takes about 7ms and the JAX version takes about 100ms.